src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void print() {
+        super.print();
+        System.out.println("right triangle,"
+                + " area: " + getArea()
+                + " sq.units,"
+                + " first leg: "
+                + firstLeg + " sq.units, "
+                + " second leg: " + secondLeg
+                + " units,"
+                + "color: " + getColor());
```

Comment:
Use only eng in messages/code: The 'sq.units' in the print statement could be misleading as it suggests square units, which is typically used for area, but here it is also used for the length of the legs. Consider using just 'units' for leg length to avoid confusion.